active_learning_single_annotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,639 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "batch_size_to_label = 100 # you can pick how many examples to collect more labels for at each round\n",
+    "\n",
+    "# get next idx to label based on batch_size_to_label and magnitude of each example's active learning score\n",
+    "next_idx_to_label = get_idx_to_label(active_learning_scores_unlabeled, batch_size_to_label=batch_size_to_label)\n",
+    "next_idx_to_label[:5],active_learning_scores_unlabeled[next_idx_to_label[:5]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Improving model accuracy over 10 rounds of active learning (collecting new labels) \n",
+    "\n",
+    "The code below shows a full demonstration of how we can repeatedly use the functions demonstrated above for multiple rounds in order to select which examples to collect labels for and use the newly collected labels to train an improved classification model.\n",
+    "\n",
+    "\n",
+    "This demonstration runs this active learning loop for 10 rounds, choosing 100 new unlabeled examples to collect more labels for each round. Each round, we use labeled examples to train a classifier (here we used autogluon's `MultiModalPredictor` classifier) and obtain predicted probabilities for the unlabeled data, which are then used to compute the active learning scores for every example. We then synthetically collect new labels (this process is meant to simulate getting annotations for a selection of examples) using `setup_next_iter_data` and repeat the active learning loop. \n",
+    "\n",
+    "[Optional step] We also measure the model performance on a test set each round to demonstrate the improvement of the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def setup_next_iter_data(df_labeled, df_unlabeled, relabel_idx_unlabeled):\n",
+    "    \"\"\"Updates inputs after additional labels have been collected in a single active learning round,\n",
+    "    this ensures that the inputs will be well formatted for the next round of active learning.\"\"\"\n",
+    "\n",
+    "    df_labeled = pd.concat([df_labeled,df_unlabeled.iloc[relabel_idx_unlabeled]], ignore_index=True)\n",
+    "    df_unlabeled = df_unlabeled.drop(relabel_idx_unlabeled)\n",
+    "    df_unlabeled = df_unlabeled.reset_index(drop=True)\n",
+    "    df_labeled = df_labeled.reset_index(drop=True)  \n",
+    "    return df_labeled, df_unlabeled"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_rounds = 5\n",
+    "batch_size_to_label = 100"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_accuacy_arr = np.full(num_rounds, np.nan)\n",
```

Comment:
typo: model_accuacy_arr

Reply:
can you also fix this typo in the active learning multiannotator notebook if it still exists? thanks!